devenv/sync.py

```diff
@@ -153,12 +125,21 @@ def main(context: dict[str, str]) -> int:
         colima.uninstall(binroot)
         limactl.uninstall(binroot)
 
+    from devenv.lib import uv
```

Comment:
instead of forcing everyone to global devenv 1.22.0 (which adds uv) i could also import guard this so the 1st sync updates local devenv to 1.22.0 then if uv is available, it can branch to calling uv sync

but by the time i remove requirements.txts (part 3) that compatibility will be for nothing, and there will be plenty of people who haven't done the first sync to update local devenv

so i'm thinking let's just keep it simple and require `devenv update`